{{cookiecutter.repo_name}}/.travis.yml

```diff
@@ -4,3 +4,17 @@ node_js:
 before_script:
   - 'npm install -g bower grunt-cli'
   - 'bower install'
+
+deploy:
+  provider: heroku
+  buildpack: python
```

Comment:
I believe this needs to be updated. The initial page rendering need to be handled by some script. Heroku doesn't support serving simple `index.html`. 

The trip i've found people use is use `php` stack. Simply add `index.php` with following content. 
```
<?php include('index.html') ?>
```